aws-cloudformation-stackset/aws-cloudformation-stackset.json

```
@@ -0,0 +1,311 @@
+{
+    "typeName": "AWS::CloudFormation::StackSet",
+    "description": "Resource Type definition for AWS::CloudFormation::StackSet",
+    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-cloudformation.git",
```

```suggestion
    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-cloudformation.git",
    "resourceLink": {
        "templateUri": "/cloudformation/home?region=${awsRegion}#/stacksets/${StackSetId}",
        "mappings": {
            "StackSetId": "/StackSetId"
        }
    },
```

https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/pull/67/

---

That's definitely good to have here. Will add in next commit.

---

@xiwhuang might need to bump this:
https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-cloudformation/blob/1e7ca3fc2d9223b7ac244ca08258991b4216e2b8/aws-cloudformation-stackset/pom.xml#L40-L45

https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/compare/v0.1.4...master

https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/compare/v2.0.2...master

https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/pull/248/files

---

actually upgrading [`cloudformation-cli`](https://github.com/aws-cloudformation/cloudformation-cli/) with `pip3 install cloudformation-cli --upgrade` solves this instead